chainer/initializers/__init__.py

```diff
@@ -74,9 +73,8 @@ def _get_initializer(initializer):
 
```

Comment:
This method is prepended with `_` but used at some `links/connections` such as `convolution_nd`.

It also creates a `Constant` initializer if the data is a chainer array, but the [`variable.Parameter`](https://github.com/chainer/chainer/blob/master/chainer/variable.py#L1683) class does a similar thing the same thing. IMHO this can be removed and the links can set the `LeCunNormal` initializer explicitly as default if no other is given?.